### ISSUE TITLE
More Stripes

### ISSUE THREAD

**Issue by credomane** [justarandomgeek/factorio-color-coding#11]
So I was playing around with the make-concrete-sprites.sh

I added the ability to generate left and right diagonals of all concrete!
Also added vertical and horizontal concrete of all colors!

Need to generation for fire-hazard concrete (will probably become a mix of red/orange instead of a bright red.

We could easily add in mixed color concretes as well (like fire-hazard potentially will be)

Created a gist for now because I deleted the wrong testing folder and lost everything. lol
Now I have to remake my diagonal and vertical masks. 
https://gist.github.com/credomane/8718dc4f41883abde49b27eb5bbde4a5

**Comment by credomane** [comment]
Updating the lua. Should I make the solid colors rotate through solid, left, vertical, right, horizontal? Or make the stripes a separate tile that rotates the stripes? 
I'm 25% for all-in-one and 75% for a solid tile then another one that rotates for the stripes.

**Comment by justarandomgeek** [comment]
Whichever one handles rotated blueprints most correctly

**Comment by credomane** [comment]
I had to split them into diagonals (the left and right variants just like vanilla hazard concrete) and stripes (Completely new vertical and horizontal striped concrete!).

Lua is updated to hopefully allow for quick adding of new tiles be them solid colors, diagonals, stripes or a specialty (fire-hazard, bio-hazard, etc)

One thing I did "break" is I renamed fire-concrete to concrete-fire-hazard-diagonal so that I could maintain full naming consistency. I added, plus tested a migration for this and it worked great.

I'll start pushing changes to a PR in a bit. Need to tweak the stripes mask for better alignment with the diagonals.
